charms/katib-controller/metadata.yaml

```diff
@@ -25,3 +25,6 @@ requires:
   k8s-service-info:
     interface: k8s-service
     limit: 1
+  logging:
```

Comment:
I just happen to notice that this was added to the `requires`, but it looks like it is more a `provides`. This charm works as a "provider" of logs so then a "requirer" (grafana agent or loki) would be able to consume.

Reply:
What we'd like to do is [this](https://github.com/canonical/grafana-agent-k8s-operator/blob/main/INTEGRATING.md#logging-consumer) right?